### ISSUE TITLE
kulttipaikat

### ISSUE THREAD

**Issue by Finto-ehdotus** [Finto-ehdotus/YSE#4035]
#### Ehdotettu termi suomeksi

kulttipaikat
#### Ehdotetut ohjaustermit

palvontapaikat
#### Ehdotetut temaattiset ryhmät (YSA-ryhmät)

[50 Kansanperinne. Perinnetieteet. Kulttuuriantropologia. Okkultismi](http://www.yso.fi/onto/ysa/ryhma_50)
[51 Arkeologia](http://www.yso.fi/onto/ysa/ryhma_51)
[53 Uskonto. Teologia. Uskontotiede](http://www.yso.fi/onto/ysa/ryhma_53)

**Luontipäivämäärä:** 16.01.2014

**Muokkauspäivämäärä:** 16.01.2014

**Voyager-id:** 196834

**Termiehdotus Fintossa:** [kulttipaikat](http://finto.fi/yse/fi/page/Y396834)

**Comment by mariaforsen** [comment]
Olisi hyvä termi kuvailuun, Melindassa jo aineistoa, esim. Making the profane sacred in the Viking age : essays in honour of Stefan Brink / edited by Irene García Losquiño, Olof Sundqvist and Declan Taggart.

RT uhripaikat ja pyhät paikat?

Sv. kultplatser
Myös SAOssa https://id.kb.se/term/sao/Kultplatser

**Comment by krinay** [comment]
Käytännössä voi olla aika hankala tehdä eroa käsitteiden pyhät paikat, uhripaikat ja kulttipaikat välille. YSOssa on lisäksi seidat. 

SAOssa offerplatser taitaa olla termin kultplatser ohjaustermi.

LCSH:ssa on ilmeisesti vain Sacred space. (Tai sitten en osannut etsiä muita.)

**Comment by ilona-x** [comment]
MAO: kulttipaikat http://www.yso.fi/onto/mao/p934

**Comment by mariaforsen** [comment]
Kyllä, SAOssa on valittu vain yksi ja silloin tietysti laajempi käsite, eli kultplatser päätermiksi. YSO on tarkempi sanasto, joten olisi luonnollista että meillä olisi molemmat erillisinä (yläkäsite ja alakäsite)

**Comment by YSO-kokous** [comment]
PÄÄTÖS

Hyväksytään YSOon kulttipaikat. 

Keskustelua: Kulttipaikka on pyhän paikan alakäsite ja uhripaikan yläkäsite. Näkökantoja esitettiin puolesta ja vastaan, mutta vaaka kallistui hyväksyntään.
